tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -33,6 +40,39 @@ Verify Content In RHODS Explore Section
     Check Number Of Displayed Cards Is Correct  expected_data=${EXP_DATA_DICT}
     Check Cards Details Are Correct   expected_data=${EXP_DATA_DICT}
 
+Verify Disabled Cards Can Be Removed
+    [Documentation]     Verifies it is possible to remove a disabled card from Enabled page.
+    ...                 It uses RHOSAK as example to test the feature
+    [Tags]    Sanity
+    ...       ODS-1081    ODS-1092
+    Enable RHOSAK
+    Remove RHOSAK From Dashboard
```

Comment:
I've called it this way to discriminate between it and `Clean Up RHOSAK` which is the full teardown which remove everything related to rhosak. 
`Remove RHOSAK From Dashboard` is the teardown used in the basic RHOSAK test where we only need to remove the card from the dashboard. I would like to keep it as it is, since I'm going to `Remove RHOSAK From Dashboard` inside the `Clean Up RHOSAK` in a refactoring step